specification/communication/data-plane/Chat/stable/2025-03-15/communicationserviceschat.json

```diff
@@ -0,0 +1,1946 @@
+            },
+            "x-ms-error-response": true
+          }
+        },
+        "x-ms-examples": {
+          "Post typing event to a thread": {
+            "$ref": "./examples/Messages_SendTypingNotification.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "CommunicationUserIdentifierModel": {
+      "$ref": "../../../Common/stable/2023-11-15/common.json#/definitions/CommunicationUserIdentifierModel"
+    },
+    "CommunicationCloudEnvironment": {
+      "$ref": "../../../Common/stable/2023-11-15/common.json#/definitions/CommunicationCloudEnvironmentModel"
+    },
+    "MicrosoftTeamsUserIdentifierModel": {
+      "$ref": "../../../Common/stable/2023-11-15/common.json#/definitions/MicrosoftTeamsUserIdentifierModel"
+    },
+    "MicrosoftTeamsAppIdentifierModel": {
+      "$ref": "../../../Common/stable/2023-11-15/common.json#/definitions/MicrosoftTeamsAppIdentifierModel"
+    },
+    "CommunicationIdentifierModel": {
+      "$ref": "../../../Common/stable/2023-11-15/common.json#/definitions/CommunicationIdentifierModel"
+    },
+    "ChatMessageReadReceipt": {
+      "description": "A chat message read receipt indicates the time a chat message was read by a recipient.",
+      "required": [
+        "chatMessageId",
+        "readOn",
+        "senderCommunicationIdentifier"
+      ],
+      "type": "object",
+      "properties": {
+        "senderCommunicationIdentifier": {
+          "$ref": "#/definitions/CommunicationIdentifierModel"
+        },
+        "chatMessageId": {
+          "description": "Id of the chat message that has been read. This id is generated by the server.",
+          "type": "string",
+          "example": "1591137790240"
+        },
+        "readOn": {
+          "format": "date-time",
+          "description": "The time at which the message was read. The timestamp is in RFC3339 format: `yyyy-MM-ddTHH:mm:ssZ`.",
+          "type": "string",
+          "example": "2020-10-30T10:50:50Z"
+        }
+      }
+    },
+    "ChatMessageReadReceiptsCollection": {
+      "description": "A paged collection of chat message read receipts.",
+      "required": [
+        "value"
+      ],
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "Collection of chat message read receipts.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ChatMessageReadReceipt"
+          }
+        },
+        "nextLink": {
+          "description": "If there are more chat message read receipts that can be retrieved, the next link will be populated.",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "CommunicationError": {
+      "$ref": "../../../Common/stable/2023-11-15/common.json#/definitions/CommunicationError"
+    },
+    "CommunicationErrorResponse": {
+      "$ref": "../../../Common/stable/2023-11-15/common.json#/definitions/CommunicationErrorResponse"
+    },
+    "ErrorResponse": {
```

Comment:
Is there a reason that you're adding the `ErrorResponse`, `Error` and `ErrorDetail` definitions? They're not being referenced and you already reference the `CommunicationErrorResponse`. Please remove them.